packages/types/src/telemetry.ts

```diff
@@ -0,0 +1,42 @@
+import type { InstanceType } from 'instance';
+
+export type TelemetryEvent = {
```

Comment:
I realized I haven't imported this within `shared/telemetry` - applying some changes here.